src/main/kotlin/com/replaymod/gradle/preprocess/PreprocessTask.kt

```diff
@@ -238,6 +241,8 @@ class CommentPreprocessor(private val vars: Map<String, Int>) {
     fun convertSource(kws: Keywords, lines: List<String>, remapped: List<Pair<String, List<String>>>, fileName: String): List<String> {
         val ifStack = mutableListOf<Boolean>()
         val indentStack = mutableListOf<Int>()
+        var trueConditionFound = false
```

Comment:
Instead of this extra variable, peeking the `ifStack` seems simpler and will work with nested ifs.